sync/class.jetpack-sync-actions.php

```diff
@@ -178,18 +182,58 @@ static function send_data( $data, $codec_name, $sent_timestamp, $queue_id ) {
 	}
 
 	static function do_initial_sync( $new_version = null, $old_version = null ) {
-		$initial_sync_config = array(
+		$initial_sync_config = self::get_update_full_sync_config();
+		$include_users = false;
+		if ( $old_version && ( version_compare( $old_version, '4.2', '<' ) ) ) {
+			$initial_sync_config['users'] = 'initial';
+			$include_users = true;
+		}
+
+		if ( is_multisite()  ) {
+			// stagger initial syncs for multisite blogs so they don't all pile on top of each other
+			if ( is_main_site() ) {
+				wp_schedule_single_event( time() , 'jetpack_sync_update_multisite_cron', $include_users );
```

Comment:
Because according to the documentation. I would have to be dealing with strange values. 
for example
```
function do_this_in_an_hour( $arg1, $arg2, $arg3 ) {
    // do something
}
add_action( 'my_new_event', 'do_this_in_an_hour', 10, 3 );

// put this line inside a function, 
// presumably in response to something the user does
// otherwise it will schedule a new event on every page visit

wp_schedule_single_event( time() + 3600, 'my_new_event', array( $arg1, $arg2, $arg3 ) );
```